pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -136,7 +136,7 @@ private static long getRandomInitialDelayInSeconds() {
   private static final String REALTIME_SEGMENT_METADATA_COMMIT_NUMLOCKS =
       "controller.realtime.segment.metadata.commit.numLocks";
   private static final String ENABLE_STORAGE_QUOTA_CHECK = "controller.enable.storage.quota.check";
-
+  private static final String ENABLE_HLC_REALTIME_SEGMENT_COMPLETION = "controller.enable.hlc.realtime.segment.completion";
```

Comment:
Can we rename the config as ALLOW_HLC_TABLES? Please document the config clearly. If the config says "true"(default),  then the behavior is unchanged. If the config  is set to "false", then the following will happen:
* Existing  HLC realtime tables will stop consumption
* Creation of mew HLC tables will be disallowed.